docs/integrations/prefect-aws/index.md

```diff
@@ -0,0 +1,24 @@
+# `prefect-aws`
```

Comment:
There's a more full-fledged index page for the docs in `prefect-aws` that's different from the README. I think we should bring that index page over in this port.

Reply:
good call! added in [65e4985](https://github.com/PrefectHQ/prefect/pull/12888/commits/65e49859571393ca2ef88ac1cf549eaeb13a99f4)